test/serverTest.js

```diff
@@ -19,462 +23,557 @@ const testLocation1Id = 'TestLocation1'
 const door_coreID = 'door_particlecoreid'
 
-function getRandomInt(min, max) {
-    min = Math.ceil(min)
-    max = Math.floor(max)
-    return Math.floor(Math.random() * (max - min) + min) //The maximum is exclusive and the minimum is inclusive
+function getRandomInt(minValue, maxValue) {
+  const min = Math.ceil(minValue)
+  const max = Math.floor(maxValue)
+  return Math.floor(Math.random() * (max - min) + min) // The maximum is exclusive and the minimum is inclusive
 }
 
 function getRandomArbitrary(min, max) {
-    return Math.random() * (max - min) + min
+  return Math.random() * (max - min) + min
 }
 
-async function silence(locationid){
-    try{
-        await chai.request(server).post('/api/xethru').send({
-            deviceid: 0,
-            locationid: locationid,
-            devicetype: 'XeThru',
-            mov_f:0,
-            mov_s: 0,
-            rpm: 0,
-            state: XETHRU_STATE.MOVEMENT,
-            distance: 0
-        })
-    }catch(e){helpers.log(e)}
+async function silence(locationid) {
+  try {
+    await chai.request(server).post('/api/xethru').send({
+      deviceid: 0,
+      locationid,
+      devicetype: 'XeThru',
+      mov_f: 0,
+      mov_s: 0,
+      rpm: 0,
+      state: XETHRU_STATE.MOVEMENT,
+      distance: 0,
+    })
+  } catch (e) {
+    helpers.log(e)
+  }
 }
 
-async function movement(locationid, mov_f, mov_s){
-    try{
-        await chai.request(server).post('/api/xethru').send({
-            deviceid: 0,
-            locationid: locationid,
-            devicetype: 'XeThru',
-            mov_f: mov_f,
-            mov_s: mov_s,
-            rpm: 0,
-            state: XETHRU_STATE.MOVEMENT,
-            distance: getRandomArbitrary(0,3)
-        })
-    }catch(e){helpers.log(e)}
+async function movement(locationid, mov_f, mov_s) {
+  try {
+    await chai
+      .request(server)
+      .post('/api/xethru')
+      .send({
+        deviceid: 0,
+        locationid,
+        devicetype: 'XeThru',
+        mov_f,
+        mov_s,
+        rpm: 0,
+        state: XETHRU_STATE.MOVEMENT,
+        distance: getRandomArbitrary(0, 3),
+      })
+  } catch (e) {
+    helpers.log(e)
+  }
 }
 
-async function door(locationid, signal){
-    try{
-        await chai.request(server).post('/api/doorTest').send({
-            deviceid: 0,
-            locationid: locationid,
-            signal: signal
-        })
-    }catch(e){helpers.log(e)}
+async function door(locationid, signal) {
+  try {
+    await chai.request(server).post('/api/doorTest').send({
+      deviceid: 0,
+      locationid,
+      signal,
+    })
+  } catch (e) {
+    helpers.log(e)
+  }
 }
 
-async function im21Door(door_coreID, signal){
-    try{
-        await chai.request(server).post('/api/door').send(
-            {
-                coreid: door_coreID,
-                data: `{ "deviceid": "FA:E6:51", "data": "${signal}", "control": "86"}`
-            })
-    }catch(e){helpers.log(e)}
+async function im21Door(coreid, signal) {
```

Comment:
Did `door_codeid` need to change because of the underscore? How about leaving the word `door` in there, so something like `doorCoreId`?

Reply:
This is just because it's "already declared in the outer scope" so that update is fine by me!

Reply:
Oh interesting. OK, I'm not too fussed. Either way is good for me. Thanks!